include/deal.II/base/function.templates.h

```diff
@@ -26,11 +26,6 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-
-template <int dim, typename Number>
-const unsigned int Function<dim, Number>::dimension;
```

Comment:
Oh, this is not a declaration but a definition. I don't think you want to remove it.